azkaban-common/src/main/java/azkaban/executor/container/ContainerCleanupManager.java

```diff
@@ -76,10 +78,13 @@
   public ContainerCleanupManager(final Props azkProps, final ExecutorLoader executorLoader,
       final ContainerizedImpl containerizedImpl,
       final ContainerizedDispatchManager containerizedDispatchManager) {
-    this.cleanupIntervalMin = azkProps
+    this.executionCleanupIntervalMin = azkProps
         .getLong(
             ContainerizedDispatchManagerProperties.CONTAINERIZED_STALE_EXECUTION_CLEANUP_INTERVAL_MIN,
-            DEFAULT_STALE_CONTAINER_CLEANUP_INTERVAL.toMinutes());
+            DEFAULT_STALE_EXECUTION_CLEANUP_INTERVAL.toMinutes());
+    this.containerCleanupIntervalMin = azkProps.getLong(
+        ContainerizedDispatchManagerProperties.CONTAINERIZED_STALE_CONTAINER_CLEANUP_INTERVAL_MIN,
```

Comment:
Would be good to sanity test this change and config on a test cluster by launching multiple fllow-pods which take a predictable amount of time to run and setting the CLEANUP_INTERVAL config to a few minutes (say 15 or so). 
Also, please explicitly test for cases the flow-pod has been deleted but the corresponding service still exists. (should be auto-deleted after these changes).